configure.ac

```diff
@@ -202,31 +200,27 @@ PKG_CHECK_MODULES(zlib,
                  [CPPFLAGS="$zlib_CFLAGS $CPPFLAGS"
                  LIBS="$zlib_LIBS $LIBS"])
 
-# Check compiler C++11 support
-AC_LANG_PUSH(C++)
-m4_define([DETECT_CPP11_PROGRAM],
-    [AC_LANG_PROGRAM([[
-#ifndef __cplusplus
-#error "This is not a C++ compiler"
-#elif __cplusplus < 201103L
-#error "This is not a C++11 compiler"
-#endif]],
-    [[]])])
-
-_tmp="$CXXFLAGS"
-CXXFLAGS="$CXXFLAGS -std=c++11"
+# Check if already in >= C++11 mode because of the flags returned by one of the above packages
+AC_MSG_CHECKING([if C++11 or later mode is enabled])
```

Comment:
maybe "if compiler defaults to C++11 or later mode"?

Reply:
@Chocobo1, @sledgehammer999: 

Perhaps, `if compiler is using C++11 or later mode`?

Reply:
@Chocobo1 that message would be slightly wrong because the mode could have been enabled by the CFLAGS of a dependency.

Probably @LordNyriox's sounds better